backend/dal/Configuration/ProjectSnapshotConfiguration.cs

```diff
@@ -15,7 +15,6 @@ public override void Configure(EntityTypeBuilder<ProjectSnapshot> builder)
             builder.ToTable("ProjectSnapshots");
 
             builder.HasKey(m => m.Id);
-            builder.Property(m => m.Id).IsRequired();
```

Comment:
An attempt to resolve the Entity Framework Bug, but for some reason it still wouldn't accept the auto generated entities from the mapper.